src/Sentry.Unity/Integrations/UnityApplicationLoggingIntegration.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using Sentry.Integrations;
+using UnityEngine;
+
+namespace Sentry.Unity.Integrations
+{
+    internal sealed class UnityApplicationLoggingIntegration : ISdkIntegration
+    {
+        private readonly IEventCapture _eventCapture;
+
+        internal readonly ErrorTimeDebounce ErrorTimeDebounce = new(TimeSpan.FromSeconds(1));
+        internal readonly LogTimeDebounce LogTimeDebounce = new(TimeSpan.FromSeconds(1));
+        internal readonly WarningTimeDebounce WarningTimeDebounce = new(TimeSpan.FromSeconds(1));
+
+        public UnityApplicationLoggingIntegration(IEventCapture eventCapture)
+        {
+            _eventCapture = eventCapture;
+        }
+
+        public void Register(IHub _, SentryOptions __)
+        {
+            Application.logMessageReceived += OnLogMessageReceived;
+            Application.quitting += OnQuitting;
+        }
+
+        private void OnLogMessageReceived(string condition, string stackTrace, LogType type)
+        {
+            var debounced = type switch
+            {
+                LogType.Error or LogType.Exception or LogType.Assert => ErrorTimeDebounce.Debounced(),
+                LogType.Log => LogTimeDebounce.Debounced(),
+                LogType.Warning => WarningTimeDebounce.Debounced(),
+                _ => true
+            };
+            if (!debounced)
+            {
+                return;
+            }
+
+            // TODO: to check against 'MinBreadcrumbLevel'
+            if (type != LogType.Error && type != LogType.Exception && type != LogType.Assert)
+            {
+                // TODO: MinBreadcrumbLevel
+                // options.MinBreadcrumbLevel
+                SentrySdk.AddBreadcrumb(condition, level: ToBreadcrumbLevel(type));
+                return;
+            }
+
+            var sentryEvent = new SentryEvent(new UnityLogException(condition, stackTrace));
+            sentryEvent.SetTag("log.type", ToEventTagType(type));
+            _ = _eventCapture?.Capture(sentryEvent);

```

Comment:
You can validate this through mocking `IHub` instead:

https://github.com/getsentry/sentry-dotnet/blob/d25c97f0e5c77b520e30877444e93a9147afb9cd/test/Sentry.Tests/AppDomainUnhandledExceptionIntegrationTests.cs#L36